src/models/simulation.jl

```diff
@@ -98,6 +98,5 @@ function finalize!(simulation::AbstractSimulation)
     unscale!(progn)                                 # undo radius-scaling for vor, div from the dynamical core
     unscale!(diagn)                                 # undo radius-scaling for vor, div from the dynamical core
     finalize!(model.output, simulation)             # possibly post-process output, then close netCDF file
-    write_restart_file!(model.output, progn)         # as JLD2 
```

Comment:
that's now part of the callbacks, that's why it's not needed as anymore, correct?

Reply:
correct